array.js

```diff
@@ -0,0 +1,11 @@
+const oldEmployees = ["Mike", "Nabil", "Cubby", "Mendes"];
```

Comment:
it looks like you have mixed up the oldEmployees with the new ones, so if u would like to change the variable names i would be pleased!
same for the row below!